src/main/scala/no/gdl/bookapi/model/domain/EditorsPick.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Part of GDL book_api.
+ * Copyright (C) 2017 Global Digital Library
+ *
+ * See LICENSE
+ */
+
+package no.gdl.bookapi.model.domain
+import no.gdl.bookapi.BookApiProperties
+import scalikejdbc._
+
+case class EditorsPick (id: Option[Long],
+                        revision: Option[Int],
```

Comment:
It is always an `Int` when reading this object from the database, but before an object of this type has been persisted, it will be `None`. Revision is added when persisting the object.